poetry.lock

```diff
@@ -1704,7 +1707,7 @@ files = [
 cffi = ">=1.4.1"
 
 [package.extras]
-docs = ["sphinx (>=1.6.5)", "sphinx-rtd-theme"]
+docs = ["sphinx (>=1.6.5)", "sphinx_rtd_theme"]
```

Comment:
I really wish I understood why this flips back and forth!

No action to take...

Reply:
Urgh, it's probably different verisons of poetry-core or something :shrug: